src/components/ShoppingTools/ShoppingTools.tsx

```diff
@@ -1,25 +1,58 @@
 import styles from './ShoppingTools.module.css';
 import { FiShoppingCart } from 'react-icons/fi';
 import { Ri24HoursLine } from 'react-icons/ri';
+import { IoIosArrowDown } from 'react-icons/io';
+import { FaFacebook } from 'react-icons/fa';
+import { RiInstagramFill } from 'react-icons/ri';
+import { AiFillTwitterCircle } from 'react-icons/ai';
+import { Link } from 'react-router-dom';
+
+import { useState } from 'react';
 
 const ShoppingTools = () => {
+  const [showInfo, setShowInfo] = useState(false);
+
   return (
     <>
       <div className={styles.contentShoppingTools}>
-        <div className={styles.support}>
+        <div
+          className={styles.support}
+          onMouseEnter={() => setShowInfo(true)}
```

Comment:
idk what's going on, but I guess you try to do something like `Tooltip` you can find some lib or read about how to make good tooltip in react